src/boot/efi/boot.c

```diff
@@ -2294,6 +2422,63 @@ static void save_selected_entry(const Config *config, const ConfigEntry *entry)
                 (void) efivar_set(LOADER_GUID, L"LoaderEntryLastBooted", NULL, EFI_VARIABLE_NON_VOLATILE);
 }
 
+static EFI_STATUS secure_boot_discover_keys(Config *config, EFI_FILE *root_dir) {
+        EFI_STATUS err;
+        _cleanup_(FileHandleClosep) EFI_FILE_HANDLE keys_basedir = NULL;
+        ConfigEntry *auto_entry = NULL;
+        SecureBootMode secure;
+
+        secure = secure_boot_mode();
+        if (!IN_SET(secure, SECURE_BOOT_SETUP))
```

Comment:
if your set has a cardinality of 1, you might as well use `!=` :P

Reply:
To be more specific (also, no need for a var here):
```suggestion
        if (secure_boot_mode() != SECURE_BOOT_SETUP)
```